cypress/plugins/index.js

```diff
@@ -0,0 +1,32 @@
+const dotenv = require("dotenv");
+/// <reference types="cypress" />
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
+// eslint-disable-next-line no-unused-vars
+dotenv.config()
+
+
+module.exports = (on, config) => {
+    // `on` is used to hook into various events Cypress emits
+    // `config` is the resolved Cypress config
+    config.env.googleRefreshToken = process.env.GOOGLE_REFRESH_TOKEN
+    config.env.googleClientId = process.env.REACT_APP_GOOGLE_CLIENTID
+    config.env.googleClientSecret = process.env.REACT_APP_GOOGLE_CLIENT_SECRET
```

Comment:
Let's stick to the existing naming convention, i.e. we already have `REACT_APP_KOKON_GSI_CLIENT_ID` and it's used in deployment configuration, so kindly please rename the other options.

Reply:
Let's discuss those variables and how can we use them here. I wasn't aware that they already existed :)